src/main/resources/templates/ccda/ccda.ftl

```diff
@@ -143,6 +143,17 @@
       <#if ehr_functional_statuses?has_content>
         <#include "functional_status.ftl">
       </#if>
+
+      <!-- 🔧 Fix for Issue #570 -->
+      <component>
```

Comment:
This component should probably be contained in its own ftl file and be included like the others, checking to make sure the attribute key has content before inclusion.